app/scripts/controllers/otherResources.js

```diff
@@ -41,6 +41,19 @@ angular.module('openshiftConsole')
       }
     });
 
+    $scope.getReturnURL = function() {
+      var kind = _.get($scope, 'kindSelector.selected.kind');
+      if (!kind) {
+        return '';
+      }
+
+      return URI.expand("project/{projectName}/browse/other?kind={kind}&group={group}", {
```

Comment:
This makes me wonder if we should be changing the URL state when you pick something in the dropdown, to make those lists bookmarkable